src/test/integration/integration-test.py

```diff
@@ -79,9 +72,7 @@ def checking_spending_limits(self, attributes):
             'spending_limit': spending_limit
         }
 
-        for key, equal_list in equal_list_dict.items():
-            logging.debug(f'{key} size: {len(equal_list)}')
-            logging.debug(f'{key} set size: {len(set(equal_list))}')
+        for equal_list in equal_list_dict.values():
```

Comment:
So, if you only wanna iterate through the values of the `equal_list_dict`, why do you want to put them into a `dict`?